apps/web/src/app.tsx

```diff
@@ -83,7 +87,15 @@ const Routing: Component = () => {
       <Route path="/onboarding" component={OnboardingPage} />
       <Route path="/members" component={MembersPage} />
       <Route path="/members/:memberId" component={MemberPage} />
-      <Route path="/requests" component={RequestsPage} />
+
+      <FeatureFlag
```

Comment:
**praised**

This is awesome. So simple yet so cool!
I add to trick by commenting and importing correct `ResquestPage` in the other branch.
Much cleaner :)